src/Formatter/CurlCommandFormatter.php

```diff
@@ -45,7 +45,12 @@ public function formatRequest(RequestInterface $request)
             } else {
                 $data = '[non-seekable stream omitted]';
             }
-            $command .= sprintf(' --data %s', escapeshellarg($data));
+
+            $escapedData = @escapeshellarg($data);
+            if (isset($php_errormsg)) {
+                throw new \InvalidArgumentException($php_errormsg);
```

Comment:
We do not want any exceptions or errors here. We want to silently fail. If we want to be extra nice to the user we may include a error message in the command. 

```php
// if fail:
$escapedData = 'We cound not escape the data properly';
```